crates/lang/ir/src/ir/attrs.rs

```diff
@@ -1116,15 +1213,15 @@ mod tests {
                 #[ink(selector = 42)]
             },
             Ok(test::Attribute::Ink(vec![AttributeArg::Selector(
-                Selector::from([0, 0, 0, 42]),
+                SelectorOrWildcard::Selector(Selector::from([0, 0, 0, 42])),
             )])),
         );
         assert_attribute_try_from(
             syn::parse_quote! {
                 #[ink(selector = 0xDEADBEEF)]
             },
             Ok(test::Attribute::Ink(vec![AttributeArg::Selector(
-                Selector::from([0xDE, 0xAD, 0xBE, 0xEF]),
+                SelectorOrWildcard::Selector(Selector::from([0xDE, 0xAD, 0xBE, 0xEF])),
```

Comment:
Maybe it would be nice to have a `From<ir::Selector>` impl for `SelectorOrWildcard` at hand.

Reply:
Or maybe even a `selector` constructor:
```rust
SelectorOrWildcard::selector([0xDE, 0xAD, 0xBE, 0xEF])
```